test/circuit_breaker_test.rb

```diff
@@ -98,6 +98,36 @@ def test_request_allowed_query_doesnt_trigger_transitions
     assert_predicate @resource, :open?
   end
 
+  def test_open_close_open_cycle
+    resource = Semian.register(:open_close, tickets: 1, exceptions: [SomeError], error_threshold: 2, error_timeout: 5, success_threshold: 2)
+
+    half_open_future_travel = -> { Time.now + resource.circuit_breaker.error_timeout + 1 }
```

Comment:
`Timecop.travel` already computes the absolute time from the delta in this [fancy method](https://github.com/travisjeffery/timecop/blob/71cbbd36ab28d95662ab980643bdeac96039b8b6/lib/timecop/time_stack_item.rb#L100-L126).